### ISSUE TITLE
Styles with 'content: "кирилица";' and config.sass.style = :compressed are raise the error "expected selector."

### ISSUE THREAD

**Issue by Sega100500** [tablecheck/dartsass-sprockets#35]
**Ruby**: 4.0.6
**Rails**: 8.1.3
gem "sprockets-rails" **3.5.2**
gem "dartsass-sprockets" **3.2.1**

```css
// this style raise error:
span.rub {
  content: " \20BD";
}

// this also raise
span.rub {
  content: "рублей";
}
```
```css
// this code solve problem
span.rub {
  --ruble-sign: " \20BD";
  content: var(--ruble-sign);
}
```

**Error raised only if ```config.sass.style = :compressed```**

Please fix it.
